_about/breaking-changes.md

```diff
@@ -41,4 +41,115 @@ A Lucene upgrade forced OpenSearch to drop support for JDK 8. As a consequence,
+- `thread_pool.test.min_queue_size`
+For more information, see [issue #2595](https://github.com/opensearch-project/OpenSearch/issues/2595).
+
+### Index store settings
+
+The `index.store.hybrid.mmap.extensions` setting has been removed as part of improvements to `hybridfs` file handling. For more information, see [pull #9392](https://github.com/opensearch-project/OpenSearch/pull/9392).
+
+### Transport Nio plugin
+
+The `transport-nio` plugin has been removed. Netty remains the standard network framework for both node-to-node and client-to-server communication. For more information, see [issue #16887](https://github.com/opensearch-project/OpenSearch/issues/16887).
+
+### Nodes API response format
+
+The format of indexing buffer values in the Nodes API response has changed:
+- `total_indexing_buffer_in_bytes` now displays raw bytes (for example, `53687091`)
+- `total_indexing_buffer` now displays human-readable format (for example, `51.1mb`)
+For more information, see [pull #17070](https://github.com/opensearch-project/OpenSearch/pull/17070).
+
+### PathHierarchy tokenizer
+
+The camel-case `PathHierarchy` tokenizer name has been deprecated in favor of the snake-case `path_hierarchy`. For more information, see [pull #10894](https://github.com/opensearch-project/OpenSearch/pull/10894).
+
+### Security plugin
+
+The Blake2b hash implementation has been fixed to correctly use the salt parameter, which will result in different (though correct) hash values compared to previous versions. For more information, see [pull #5089](https://github.com/opensearch-project/security/pull/5089).
+
+### k-NN plugin
+
+The following deprecated settings have been removed from the k-NN plugin:
+
+- `knn.plugin.enabled` setting
+- `index.knn.algo_param.ef_construction` index setting
+- `index.knn.algo_param.m` index setting
+- `index.knn.space_type` index setting
+
+The NMSLIB engine is now deprecated. We recommend using the Faiss or Lucene engines instead.
+
+For more information, see [pull #2564](https://github.com/opensearch-project/k-NN/pull/2564).
+
+### Performance Analyzer plugin
+
+The `performance-analyzer-rca` agent has been removed. We recommend transitioning to the [Telemetry plugin](https://github.com/opensearch-project/performance-analyzer/issues/585) for performance monitoring and analysis. The Telemetry Plugin, using the OpenTelemetry framework, allows for seamless integration with lightweight open-source agents to publish performance metrics to observability stores. For more information, see [issue #591](https://github.com/opensearch-project/performance-analyzer-rca/issues/591).
+
+### SQL plugin
+
+- OpenSearch query DSL response format has been removed.
+- `DELETE` statement support has been removed.
+- The `plugins.sql.delete.enabled` setting has been removed.
+- SparkSQL support has been removed from the OpenSearch Spark connector.
+- Deprecated OpenDistro endpoints and Legacy settings with the `opendistro` prefix have been removed.
```

Comment:
Overall LGTM for SQL plugin. Small comments:

1. `SparkSQL support has been removed from the OpenSearch Spark connector` -> `Deprecated legacy Spark Connector Module` 
2. `- Deprecated OpenDistro endpoints and Legacy settings with the `opendistro` prefix have been removed.
` nit: `Legacy ` -> `legacy` ?